src/css/popup.css

```diff
@@ -578,6 +578,11 @@ span ~ .panel-header-text {
   max-inline-size: 204px;
 }
 
+.disable-edit-containers {
+    opacity: 0.3;
```

Comment:
It seems like we use `opacity: 0.3;` in a few places now too, it probably makes sense to move that out to it's own custom property at this point.

```
:root {
  --inactive-opacity: 0.3;
}
.thing-that-is-disabled {
  opacity: var(--inactive-opacity);
}
```